docs/hardware/common.md

```diff
@@ -0,0 +1,223 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Hardware Common Metrics
+--->
+
+# Semantic Conventions for Hardware Common Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+All metrics in `hw.` instruments should be attached to a [Host Resource](/docs/resource/host.md)
+and therefore inherit its attributes, like `host.id` and `host.name`.
+
+Additionally, all metrics in `hw.` instruments have the following attributes:
+
+<!-- semconv hardware.attributes.common -->
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`hw.id`](/docs/attributes-registry/hw.md) | string | An identifier for the hardware component, unique within the monitored host | `win32battery_battery_testsysa33_1` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`hw.name`](/docs/attributes-registry/hw.md) | string | An easily-recognizable name for the hardware component | `eth0` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`hw.parent`](/docs/attributes-registry/hw.md) | string | Unique identifier of the parent component (typically the `id` attribute of the enclosure, or disk controller) | `dellStorage_perc_0` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+## Common hardware metrics
+
+The below metrics apply to any type of hardware component.
+
+These common `hw.` metrics include the below attributes to describe the
+monitored component:
+<!-- semconv metric.hardware.attributes(full) -->
```

Comment:
we have another attribute table defined above, which of them should be applied to metrics?

Reply:
In original md file here was defined only `type` 
common attributes extend the table above therefore when you render it it has both previous attributes and `type`. Do you know how to skip it or should we skip it?

Reply:
Since each metric in this file defines extra attributes in addition to common ones, I suggest removing common attributes and just generating full attribute tables for each metric.

Otherwise it becomes hard to understand from the reader perspective which attribute apply to a specific metric without scrolling back and forth

Reply:
I agree with @lmolkova's suggestion here.